src/styles/blocks/stars--4.css

```diff
@@ -0,0 +1,44 @@
+.stars--4 {
+  display: flex;
+  margin-top: 4px;
+}
+
+.stars--4 .stars__star:nth-child(1) {
+  display: block;
+  background-image: url(../../images/star-active.svg);
+  width: 13px;
+  height: 12.2px;
+  margin-left: 22px;
```

Comment:
The first star has a different `margin-left` value compared to the others. Ensure this is intentional. If all stars are supposed to be evenly spaced, this value should be consistent with the margin-left of other stars.